control-plane/csi-driver/src/bin/node/nodeplugin_nvme.rs

```diff
@@ -23,14 +81,20 @@ impl NvmeOperations for NvmeOperationsSvc {
             Some(map_wrapper) => map_wrapper.map,
             None => HashMap::new(),
         };
-        // Get the nvmf device object from the uri
+
+        // Create a new request entry, to store in request inventory.
+        let entry = RequestEntry::new(uri.to_string());
+        // Insert the entry item to inventory, error out if already present.
+        entry.insert()?;
```

Comment:
could this be single operation? ex:
let _guard = VolumeOpGuard::new(&uri)?;

Reply:
Yep.

Reply:
Done.